mongodbatlas/resource_mongodbatlas_advanced_cluster.go

```diff
@@ -714,7 +715,7 @@ func resourceMongoDBAtlasAdvancedClusterUpdate(ctx context.Context, d *schema.Re
 	timeout := d.Timeout(schema.TimeoutUpdate)
 
 	// Has changes
-	if !reflect.DeepEqual(cluster, matlas.Cluster{}) {
+	if !reflect.DeepEqual(cluster, clusterChangeDetect) {
```

Comment:
do we need to make similar change in mongodb_cluster.go as well?

Reply:
Added to cluster resource to avoid extra update call with no changes